add some activities to handbook (#657)
CodeYourFuture/curriculum: common-content/en/activities/in-your-own-words/_index.md

+++
title="In your own words"

> description="You’ll have to take an existing term and then explain it in their own words"

Reviewer: ```suggestion
description="Take an existing term and then explain it in your own words"
```
